webapp/src/main/webapp/src/assets/i18n/en.json

```diff
@@ -105,6 +107,16 @@
       "MATH": "Math",
       "ELA": "ELA",
       "ALL": "All"
+    },
+    "status": {
+      "UNKNOWN_SCHOOL": "Unknown School",
+      "UNKNOWN_ASMT": "Unknown Assessment",
+      "UNAUTHORIZED": "Unauthorized",
+      "BAD_DATA": "Bad Data",
+      "BAD_FORMAT": "Bad Format",
+      "INVALID": "Invalid",
+      "ACCEPTED": "Accepted",
+      "PROCESSED": "Processed"
```

Comment:
@wtritch @chdietz @rcordeau  I realize the status enum was just a copy / paste from Ingest.  Should you add / remove statuses please update their translate values here.